packages/fr-config-push/src/helpers/config-process.js

```diff
@@ -38,3 +45,4 @@ function removeProperty(obj, propertyName) {
 
 module.exports.replaceEnvSpecificValues = replaceEnvSpecificValues;
 module.exports.removeProperty = removeProperty;
+module.exports.unescapePlaceholders = unescapePlaceholders;
```

Comment:
Instead of exporting the unescapePlaceholders function, call it after replacing the non-escaped placeholders. then the function call stays the same

Reply:
Should the unescape function be extracted in to `utils.js` along with the escapePlaceholders function? Seems odd to have them in two separate places

Reply:
I have moved `unescapePlaceholders` into `replaceEnvSpecificValues` so that it's a single function call.

Also put the escape and unescape functions together in `utils.js` and moved `utils.js` to `fr-config-common` so it is accessible for pull and push packages.